templates/cloudfront-site.yaml

```diff
@@ -87,6 +94,8 @@ Resources:
       DistributionConfig:
         Aliases:
           - !Sub '${SubDomain}.${DomainName}'
+          - !If [ CreateApexConfig, !Ref DomainName, !Ref 'AWS::NoValue' ]
+#          - !Ref DomainName
```

Comment:
Should this still be here?

Reply:
This is intended.   When the [CreateApexConfig] condition is met, CloudFront CNAMEs for both apex-domain **and**  sub-domain are created within a distribution

So given the following  parameters ...

- SubDomain = www
- DomainName = example.com
- CreateApexConfig = yes

... The CloudFront CNAMEs below are created

- example.com # apex-domain
- www.example.com # sub-domain

Reply:
I meant the commented-out line specifically, since it doesn't appear to be a comment but some unused implementation.

Reply:
Missed that .... commented out properties have been removed

Reply:
> This is intended. When the [CreateApexConfig] condition is met, CloudFront CNAMEs for both apex-domain **and** sub-domain are created within a distribution
> 
> So given the following parameters ...
> 
> * SubDomain = www
> * DomainName = example.com
> * CreateApexConfig = yes
> 
> ... The CloudFront CNAMEs below are created
> 
> * example.com # apex-domain
> * [www.example.com](http://www.example.com) # sub-domain

The fact that SubDomain should still be "www" when CreateApex is true needs to be much better documented! I fumbled around with this for 30 minutes before finding this comment in the PR.

Also, it does not seem like acm-certificate.yaml has been uploaded to the template yet. The URL in the readme does not point to a version that contains CreateApex.